UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/TileChangeManager.cs

```diff
@@ -99,10 +100,151 @@ void LoadAllTileAssets ()
 		}
 		init = true;
 	}
+
+	[Server]
+	public void ChangeTile (string newTileName, Vector2Int tileCellPos, TileChangeLayer layer)
+	{
+		ActionChange (new TileChangeEntry
+		{
+			cellPosition = tileCellPos,
+				tileKey = newTileName,
+				layerToChange = layer
+		});
+	}
+
+	[Server]
+	public void RemoveTile (Vector2Int tileCellPos, TileChangeLayer layer)
+	{
+		ActionChange (new TileChangeEntry
+		{
+			cellPosition = tileCellPos,
+				tileKey = "",
+				layerToChange = layer
+		});
+	}
+
+	private void ActionChange (TileChangeEntry changeEntry)
+	{
+		if (!ValidateChange (changeEntry.layerToChange, changeEntry.tileKey))
+		{
+			Logger.LogError ("No key found for Tile Change");
+			return;
+		}
+
+		var tileMap = GetTilemap (changeEntry.layerToChange);
+		TileBase newTile = GetTile(changeEntry.layerToChange, changeEntry.tileKey);
+		tileMap.SetTile(new Vector3Int(changeEntry.cellPosition.x,
+		changeEntry.cellPosition.y, 0), newTile);
+
+		if(isServer){
+			RpcActionChange(changeEntry.cellPosition, changeEntry.tileKey, changeEntry.layerToChange);
+		}
+		//TODO ADD TO THE CHANGE REGISTER
+	}
+
+	[ClientRpc]
+	private void RpcActionChange(Vector2 cellPos, string tileKey, TileChangeLayer layer){
```

Comment:
Using ClientRpc will lead to inconsistencies on client side (some tiles will not be updated), keep that in mind

Reply:
it's on the default reliable channel so it should work. Will keep an eye on it